include/rtc/peerconnection.hpp

```diff
@@ -113,6 +118,7 @@ class RTC_CPP_EXPORT PeerConnection final : CheshireCat<impl::PeerConnection> {
 	void onSignalingStateChange(std::function<void(SignalingState state)> callback);
 
 	void resetCallbacks();
+	std::vector<struct RemoteFingerprint> remoteFingerprints();
```

Comment:
```suggestion
	std::vector<CertificateFingerprint> remoteFingerprints();
```

Reply:
Given there's only every going to be one of these we should probably just make this:

```c++
CertificateFingerprint remoteFingerprint();
```

Reply:
Done!